### PR TITLE
Display the title of the tutorial

### PR DESCRIPTION
**Pull request by e111077**
See in the tutorial section

### CODE REVIEW

File: packages/lit-dev-content/src/components/litdev-tutorial.ts [hunk 1]
```diff
@@ -261,9 +263,12 @@ export class LitDevTutorial extends LitElement {
...
-        <span>
+        <span class="tutorial-metadata">
           ${this._manifestTask.render({
-            complete: () => html`Step
+            complete: (manifest) => html`
+              <span class="tutorial-title">${manifest?.header}</span>
+              <span class="title-separator">${diamondIcon}</span>
+              <span class="step-text">Step</span>
               <span class="number">${this._idx + 1}</span>
               / <span class="number">${this._manifest.steps.length}</span>`,
```
Comment (AndrewJakubowicz) [inline packages/lit-dev-content/src/components/litdev-tutorial.ts:273]: Here usage is `this._manifest.steps`. Should this also be `manifest?.steps`?

Comment (e111077) [reply packages/lit-dev-content/src/components/litdev-tutorial.ts:274]: it doesn't really matter here but thanks for pointing that out. Can probably clean this up

File: packages/lit-dev-content/site/css/tutorial.css [hunk 2]
```diff
@@ -107,6 +132,7 @@ litdev-tutorial {
   /* TODO(e111077): I don't know why this nav will not encapsulate the width
   of the buttons inside of it otherwise */
   width: 96px;
```
Comment (AndrewJakubowicz) [inline packages/lit-dev-content/site/css/tutorial.css:134]: Regarding this TODO - There is a `margin-right: -20px` on `#tutorialHeader #nextButton`. Removing that margin-right allows the removal of this width - I think.

**Review by AndrewJakubowicz — APPROVED** [review]
LGTM!

File: packages/lit-dev-content/site/css/tutorial.css (hunk 2 above)
Comment (AndrewJakubowicz) [reply packages/lit-dev-content/site/css/tutorial.css:134]: Hard to find a good resource. But found some whispers that negative margins behave differently with and without a width specified.

Comment (e111077) [reply packages/lit-dev-content/site/css/tutorial.css]: PHENOMENAL find!

**Pull request merged**